api/School.php

```diff
@@ -174,4 +177,35 @@ function where($params) {
 
 		return $returnCollection;
 	}
+
+	function get($params) {
+		$records = $this->conn->get('schools', $params);
+		$returnCollection = new Collection([]);
+		if(count($records) > 0) {
+			foreach ($records as $key => $record) {
+				$tmp = new School($this->conn);
+				$tmp->create($record);
+				$departments = $this->conn->get('school_has_department', ['school_id' => $record['id']]);			
+				$departments_plucked = array_column($departments, 'department_id');
+				$tmp->update(['departments' => $departments_plucked]);
+				$tmpcity = $this->conn->get('cities', ['id' => $tmp->city]);
+				$tmpcountry = $this->conn->get('countries', ['id' => $tmp->country]);
+				
+				$fixedtmp = array(
+					"id" => $tmp->id,
+					"name" => $tmp->name,
+					"country" => $tmpcountry[0]['name'],
+					"city" => $tmpcity[0]['name'],
+					"place_id" => $tmp->place_id,
+					"departments" => $tmp->departments,
+					);
+				$returnCollection->push($fixedtmp);
+			}
+		} else {
+			return NULL;
```

Comment:
Kommentoin äsken väärän paikkaan eli tänne uusiks 😄 Tää on siis joo parempi näin eli vois poistaa ton alemman rivin Exceptionin heiton?